src/components/PageWithSideMenu.tsx

```diff
@@ -76,22 +78,38 @@ interface LinkButtonProps extends ButtonProps, Partial<Pick<LinkProps, 'to'>> {
 
 export const LinkButton = ({ isSelected, sx, ...rest }: LinkButtonProps) => (
   <Button
-    sx={{ bgcolor: isSelected ? 'primary.main' : 'background.default', ml: '1rem', width: 'fit-content', ...sx }}
+    sx={{ bgcolor: isSelected ? 'primary.main' : 'background.default', justifyContent: 'start', ...sx }}
     variant={isSelected ? 'contained' : 'outlined'}
     LinkComponent={rest.to ? Link : undefined}
     {...rest}
   />
 );
 
 interface LinkIconButtonProps extends LinkButtonProps {
-  icon: ReactNode;
+  selectedColor?: string;
 }
 
-export const LinkIconButton = ({ sx, icon, ...rest }: LinkIconButtonProps) => (
-  <LinkButton sx={{ ml: '0rem', mr: '1rem', ...sx }} {...rest}>
-    {icon}
-  </LinkButton>
-);
+export const LinkIconButton = ({ sx, title, isSelected, selectedColor, ...rest }: LinkIconButtonProps) => {
+  const { t } = useTranslation();
+  return (
+    <LinkButton
+      sx={{
```

Comment:
Litt innfløkt dette, mtp at `LinkButton` også har en del custom styling. Så vi arver i 3 steg: `Button` -> `LinkButton` ->`LinkIconButton`, hvor jeg i hvert fall syns det blir vanskelig å holde styr på hvilken stil som kommer fra hvor, og gjør det litt mer knot med gjenbruk. Tror likevel akkurat det får være greit for nå, så skal jeg lufte problemet med at vi har mange ulike typer knapper, og at vi ikke har helt kontroll på hvordan de skal se ut med hover etc i (kanskje) neste sprint. Må uansett rette opp i en del ting da